src/int.rs

```diff
@@ -28,6 +49,37 @@ enum Sign {
 }
 
 impl I256 {
+    /// Maximum value.
+    pub const MAX: I256 = I256(U256([u64::MAX, u64::MAX, u64::MAX, i64::MAX as _]));
```

Comment:
Interesting. Is this really easier than let's say 2**255-1?

Reply:
No :cry: , but necessary to make it a `const`. We would have to use something like `lazy_static` otherwise.

Reply:
I have a unit test to make sure the value is correct. :)